integration/helpers/deno-template/.eslintignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Could we add a comment explaining why this file is here?

Reply:
Yea I think we can just remove this file and be ok.